### DISCUSSION TITLE
[Feature Request] Add "Edit and Continue" feature for debugging

### DISCUSSION THREAD

**Discussion by Megil-Zesty** [microsoft/debugpy#1260] [Enhancements]
For the love of god, please find a way to run an "Edit-and-continue" workflow in the debugger for Python. I realize others have said Python doesn't support it, but there must be a way...

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Comment by fabioz** [comment]
@Megil-Zesty @judej there is already support for edit and continue in debugpy. It does have some caveats (for instance, if you edit the function of the frame you're currently in you need to exit the frame for it to get the changes because in CPython the frame is immutable), but the basics work.

See: https://github.com/microsoft/debugpy/issues/1119#issuecomment-1310212115 for info on using it.
